### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by yxliang01** [dtao/lazy.js#225]
From #216 and #161, it looks like this project is still maintained by the original author @dtao . Just that it's not so active. So, I am just wondering, after many years of development, is this project stable now? Is the last sentence added 6 years ago in the README ("This library is experimental and still a work in progress.") still reflecting the current status?

Thanks

**Comment by dtao** [comment]
Fair question :)

I think this library is pretty stable. It's got lots of [dependent projects](https://www.npmjs.com/browse/depended/lazy.js). (I'm not sure how active _they_ all are, of course.)

With that in mind, I should probably remove that note about the library being experimental from the README.

Aside from the library being _stable_, there are plenty of _improvements_ I would like to make, if I had unlimited time. For what it's worth, I think I have generally made at least a modest effort to fix actual _bugs_ when they are raised. Last I checked, most of the open issues today are requests for new features, or for existing features to work a bit differently.
